docs/NSR.md

```diff
@@ -144,7 +144,9 @@ f.	Do one of the following within ninety-six (96) hours of discovery of a Critic
 
 **Multi-Factor Authentication:** An authentication mechanism consisting of two or more of the following independent categories of credentials (i.e. factors) to verify the user’s identity for a login or other transaction: something you know (knowledge factor), something you have (possession factor), and something you are (inherence factor). Each factor must be independent. Certificate-based authentication can be used as part of Multifactor Authentication only if the private key is stored in a Secure Key Storage Device.
 
-**National Vulnerability Database (NVD):**   A database that includes the Common Vulnerability Scoring System (CVSS) scores of security-related software flaws, misconfigurations, and vulnerabilities associated with systems (see http://nvd.nist.gov/home.cfm). 
+**National Vulnerability Database (NVD):**   A database that includes the Common Vulnerability Scoring System (CVSS) scores of security-related software flaws, misconfigurations, and vulnerabilities associated with systems (see http://nvd.nist.gov/home.cfm).
+
+**Offline Root CA System:** A Root CA System that is operated in a non-networked, stand-alone state and powered down when not in use.
```

Comment:
Air-gapped: Physically segregated (NOTE: it is not sufficient for logical segregation alone (e.g., firewall rules, VLANs, etc.)

Offline: Stand-alone or networked computer system that is maintained in a powered off state when not actively in use.

Non-Networked - System that is only accessible from a console, and cannot access any resources outside of those directly connected to the system.